### PR TITLE
Clarification of return values descriptions for mysqi fetch functions

### PR DESCRIPTION
**Pull request by afilina**
I fixed the merge conflicts between the two PRs #39 and #207 as best I could. @colshrapnel did not enable changes from maintainers, so I had to open a new PR instead of appending to the existing one.

Closes #39

### CODE REVIEW

File: reference/mysqli/mysqli_result/fetch-row.xml [hunk 1]
```diff
@@ -40,7 +40,7 @@
  <refsect1 role="returnvalues">
   &reftitle.returnvalues;
   <para>
-   <function>mysqli_fetch_row</function> returns an array of values that corresponds to the fetched row
+   <function>mysqli_fetch_row</function> returns an enumerated array representing the fetched row
```
Comment (cmb69) [inline reference/mysqli/mysqli_result/fetch-row.xml:43]: I think the term "indexed array" is more common elsewhere in the manual, but the MySQLi docs appear to prefer "enumerated array", so this is fine.

File: reference/mysqli/mysqli_result/fetch-object.xml [hunk 2]
```diff
@@ -67,10 +67,18 @@
...
-   Returns an object that corresponds to the fetched
-   row or &null; if there are no more rows in result set.
+   Returns an object representing the fetched row, where each property
+   represents the name of the result set's column or &null;
+   if there are no more rows in the result set.
   </para>
- </refsect1>
+  <para>
+   If two or more columns of the result have the same name, the last
+   column will take precedence. To access the other column(s) of the same
```
Comment (kamil-tekiela) [inline reference/mysqli/mysqli_result/fetch-object.xml:76]: I am not convinced about "the other" as a determiner in this sentence here. (see https://dictionary.cambridge.org/grammar/british-grammar/other-others-the-other-or-another) but I am not a native speaker, so maybe this is ok? However, this is how it is currently written on the fetch_assoc page.

**Review by kamil-tekiela — COMMENTED** [review]
The changes look good, but I am only concerned about the ease of reading. I think the comma should be placed there to separate the thoughts in the sentence, and if someone could confirm whether "the other" is correct as the determiner in this case, it would be great.

File: reference/mysqli/mysqli_result/fetch-object.xml (hunk 2 above)
Comment (afilina) [reply reference/mysqli/mysqli_result/fetch-object.xml:76]: In this case, "the other" refers to a set that excludes the element previously mentioned, so this looks ok to me.

**Review by kamil-tekiela — APPROVED** [review]

**Comment by kamil-tekiela** [comment]
Can you please add "Co-authored-by" with the original author name, please?

File: reference/mysqli/mysqli_result/fetch-object.xml (hunk 2 above)
Comment (salathe) [reply reference/mysqli/mysqli_result/fetch-object.xml:76]: This paragraph (when viewed as a whole) is misplaced in the "Return Values" section.  

The "Return Values" section should not contain any direction about what to do if the described method or function does not do what the reader wants: it should stick to describing the return values.  

Directing readers to other, more appropriate for their needs, functions should be in a "Note" (best for short, to-the-point notes)  or a paragraph in the "Description" section.

Comment (afilina) [reply reference/mysqli/mysqli_result/fetch-object.xml:76]: @colshrapnel ^

Comment (pthreat) [reply reference/mysqli/mysqli_result/fetch-object.xml]: "remaining" instead of the other would be better

**Review by salathe — APPROVED** [review]

**Pull request merged**